Program.cs

```diff
@@ -43,9 +41,8 @@ public sealed partial class App
         public static IDAL CosmosDal { get; set; }
 
         public static string CosmosName { get; set; } = string.Empty;
-        public static bool UseCache => Cache || Ngsa.Middleware.RequestLogger.RequestsPerSecond > Constants.MaxReqSecBeforeCache;
 
-        public static Config Config { get; } = new Config();
+        public static Config Config { get; set; } = new Config();
```

Comment:
Question, since Config member gets assigned inside CommandLine.RunApp function, do we still need to initialize it at the partial class when defining the property ? 
Seem like this will give us a Config instance with default values for a short period of time until App actually starts Running. I do not see any other references to is meanwhile, so it might not be a big deal, but if there is a code refactoring later at least a null value will throw an exception rather the use default. Just a thought

Reply:
good catch - resolved